framework/src/Volo.Abp.MongoDB/Volo/Abp/Domain/Repositories/MongoDB/MongoDbRepositoryFilterer.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Collections.Generic;
+using MongoDB.Driver;
+using Volo.Abp.Data;
+using Volo.Abp.Domain.Entities;
+using Volo.Abp.MultiTenancy;
+
+namespace Volo.Abp.Domain.Repositories.MongoDB
+{
+    public class MongoDbRepositoryFilterer<TEntity> : IMongoDbRepositoryFilterer<TEntity>
```

Comment:
We can make the method in the implementation class **virtual**.

Reply:
Done.